subsys/nfc/t4t/isodep.c

```diff
@@ -156,9 +156,10 @@ static size_t did_include(u8_t *data, size_t pos)
 		data[pos] |= I_BLOCK_DID_BIT;
 		pos++;
 		data[pos] = t4t_isodep.tag.did;
-		pos++;
 	}
 
+	pos++;
```

Comment:
why? no data was written?

Reply:
data[pos] is considered ready at that point, because NAD bit presence is determined in this function.